modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -418,6 +418,7 @@ export default class Blogs {
 						referringPortletResource: this._getElementById(
 							'referringPortletResource'
 						).value,
+						isCreationScreen: this._getElementById('isCreationScreen').value,
```

Comment:
Instead of calling this `isCreationScreen`, can you rename it so that its purpose is clear? E.g. `showPermissions`.